doc/source/getting-started/administration.rst

```diff
@@ -118,7 +118,8 @@ The following checks are required when performing the legal review of the projec
     * Ansys official logos and branding images are used in the project.
     * The Ansys copyright appears in the correct location as required by the
       Legal department.
-    * The copyright has the proper formatting, which is ``Copyright (C) YYYY ANSYS, Inc.``.
+    * The copyright has the proper formatting, which is:
+      ``Copyright (C) YYYY ANSYS, Inc. and/or its affiliates.``.
```

Comment:
I would ask users to implement the pre-commit hook rather than specifying the sentence we need.

Reply:
I would keep that sentence because, it is just part of a checklist.
We are not asking people to write it on their own. But we are just checking that copyright appears as expected, part of the checklist. Regardless of the way it was generated and the tool used.

Then, we can advise somewhere else in the dev-guide to make an extensive usage of the pre-commit hook.

Reply:
Sounds good